csharp/benchmark/SeedLang.Benchmark/Program.cs

```diff
@@ -12,19 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace SeedLang.Runtime {
-  public enum ValueType {
-    Number,
-    String
-  }
+using BenchmarkDotNet.Running;
 
-  // The value interface of all value types.
-  //
-  // There are different kinds of value types in the AST and interpreter components. They all
-  // implement this value interface.
-  public interface IValue {
-    ValueType Type { get; }
-    double ToNumber();
-    string ToString();
+namespace SeedLang.Benchmark {
+  class Program {
+    static void Main(string[] _) {
+      BenchmarkRunner.Run<BinaryExpressionBenchmark>();

```

Comment:
Suggestion: we have a benchmark program now. It's great to track the benchmark result every time we improve the codebase. Other than a decent automatic tracking system (e.g., run the benchmark in CI and record the result somewhere automatically), we can at least create a separate GitHub issue to manually log those results - simply comment on the issue each time we have made a non-trivial code change.

Of course, my suggestion doesn't block this pull request at all.